src/app/components/media/MediaActionsBar.js

```diff
@@ -103,7 +103,17 @@ class MediaActionsBarComponent extends Component {
   }
 
   currentProject() {
-    return this.props.media.project;
+    let project = null;
+    let currentProjectId = window.location.pathname.match(/project\/([0-9]+)/);
+    if (currentProjectId) {
+      currentProjectId = parseInt(currentProjectId[1], 10);
+      project = this.props.media.projects.edges.filter(p =>
+        parseInt(p.node.dbid, 10) === currentProjectId);
+      if (project.length) {
+        project = project[0].node;
+      }
+    }
+    return project;
```

Comment:
Looks like this should be a helper function for better reusability

Reply:
Yes Alex I added it to helper and another one to just get the projectId and depend on `project_ids` Array not whole projects